examples/input/mouse_input.rs

```diff
@@ -1,14 +1,29 @@
 //! Prints mouse button events.
 
-use bevy::prelude::*;
+use bevy::{
+    input::mouse::{AccumulatedMouseMotion, AccumulatedMouseScroll},
+    prelude::*,
+};
 
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
-        .add_systems(Update, mouse_click_system)
+        .init_resource::<CurrentMouseMotion>()
+        .init_resource::<CurrentMouseScroll>()
+        .add_systems(Update, (mouse_click_system, mouse_move_system))
         .run();
 }
 
+#[derive(Resource, Default)]
+struct CurrentMouseMotion {
```

Comment:
I don't understand the point of these resources: aren't they just doing what the accumulated forms should already be doing?

Reply:
The resources track the final position of the motion, so that when you swipe you don't get a ton of debug messages showing how much the mouse moved in that frame, and instead you only get a message of the final position of the mouse after the swipe

Reply:
(The same applies with scrolling)

Reply:
I guess the whole "spamming the console" thing is done by the clicking portion of the example anyway, so I could make the example shorter by just printing out the accumulated mouse inputs whenever they're not zero

Reply:
Yeah, I think that will be clearer :)